src/store/UI_store/CurrencySlice/CurrencySlice.ts

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+//types
+import { ICurrencyState } from './CurrencyInterfaces';
+
+const initialState: ICurrencyState = {
+    currency: '₴'
+}
+
+export const CurrencySlice = createSlice({
+    name: 'CurrencySlice',
+    initialState,
+    reducers: {
+        setCurrency: (initialState): void => {
+            initialState.currency = (initialState.currency === '₴') ? '$' : '₴'
```

Comment:
and here we should set like setCurrency: (initialState, action: PayloadAction<ICurrency>): void => {
            initialState.currency = (initialState.currency === '₴') ? '$' : '₴'
            
            type ICurrency = '' | '' | '' | '' | ''